__tests__/app.test.js

```diff
@@ -52,18 +52,89 @@ describe("GET /api/articles/article_id", () => {
             topic: "mitch",
             author: "butter_bridge",
             body: "I find this existence challenging",
-            created_at: '2020-07-09T20:11:00.000Z',
+            created_at: "2020-07-09T20:11:00.000Z",
             votes: 100,
           })
         );
       });
   });
-  test('status: 400 responds when given article is not a valid one', () => {
-        return request(app)
-        .get('/api/articles/notAnId')
-        .expect(400)
-        .then(({ body: { msg }}) => {
-          expect(msg).toBe('Invalid input')
-        })
+  test("status: 400 responds when given article is not a valid one", () => {
+    return request(app)
+      .get("/api/articles/notAnId")
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Invalid input");
+      });
+  });
+});
+
+// --- PATCH articles ---
+describe("PATCH /api/articles/:article_id", () => {
+  test("status: 201 responds with article object featuring added votes", () => {
+    return request(app)
+      .patch("/api/articles/7")
+      .send({ inc_votes: 50 })
+      .expect(201)
```

Comment:
I would actually suggest a good old 200 is actually more appropriate here. 201 "Created" is more often associated with the creation of a brand new resource rather than an amendment to an existing one